site/src/measure.njk

```diff
@@ -0,0 +1,165 @@
+---
+title: Quicklink
+layout: layouts/base.njk
+description: Faster subsequent page-loads by prefetching in-viewport links during idle time.
+---
+{% extends "layouts/normal-section-wrapper.njk" %}
+{% block section %}
+{% markdownConvert %}
+## Measuring impact of QuickLink in sites
+
+Implementing Quicklink in sites can speed up navigations, by automatically prefetching in-viewport links during idle time.
+Different metrics can be improved as a result of this, the most common ones being [Start Render](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/quick-start-quide#TOC-Start-Render:) and [First Contentful Paint](https://developers.google.com/web/tools/lighthouse/audits/first-contentful-paint).
+
+In this section, we explore different ways of measuring the impact of Quicklink in sites. To showcase that, we’ll use [this demo](https://mini-ecomm.glitch.me/), consisting of a listing and product page. We'll compare performance of that site with [this other version](https://mini-ecomm-quicklink.glitch.me/) that requests and initializes the library in the product listing page. You can check the head of the page, to see the following code before the closing tag:
```

Comment:
Could we expand `this demo` to link a link to the demo as well as a source link on Glitch for folks that want to fork it or view source?